janus/models/workflow/balsamic.py

```diff
@@ -31,5 +31,5 @@ class BalsamicSample(BaseModel):
 
 class Balsamic(BaseModel):
     samples: list[BalsamicSample]
-    somalier: Somalier | None
+    somalier: Somalier | None = Field(None)
```

Comment:
Since we're not providing additional metadata

```suggestion
    somalier: Somalier | None = None
```